base/shell/explorer/trayntfy.cpp

```diff
@@ -189,25 +225,122 @@ class CTrayNotifyWnd :
         return TRUE;
     }
 
-    VOID Size(IN const SIZE *pszClient)
+    VOID Size(IN OUT SIZE *pszClient)
     {
+        RECT rcClient = {0, 0, pszClient->cx, pszClient->cy};
+        AlignControls(&rcClient);
+        pszClient->cx = rcClient.right - rcClient.left;
+        pszClient->cy = rcClient.bottom - rcClient.top;
+    }
+
+    VOID AlignControls(IN PRECT prcClient OPTIONAL)
+    {
+        RECT rcClient;
+        if (prcClient != NULL)
+        {
+            rcClient.left = prcClient->left;
+            rcClient.top = prcClient->top;
+            rcClient.right = prcClient->right;
+            rcClient.bottom = prcClient->bottom;
+        }
+        else
+        {
+            if (!GetClientRect(&rcClient))
+            {
+                ERR("Could not get client rect lastErr=%d\n", GetLastError());
+                return;
+            }
+        }
+
+        rcClient.left += ContentMargin.cxLeftWidth;
+        rcClient.top += ContentMargin.cyTopHeight;
+        rcClient.right -= ContentMargin.cxRightWidth;
+        rcClient.bottom -= ContentMargin.cyBottomHeight;
+
+        UINT swpFlags = SWP_DRAWFRAME | SWP_NOCOPYBITS | SWP_NOZORDER;
+
+        if (g_TaskbarSettings.bShowDesktopButton)
+        {
+            POINT ptShowDesktop =
+            {
+                rcClient.left,
+                rcClient.top
+            };
+            SIZE szShowDesktop =
+            {
+                rcClient.right - rcClient.left,
+                rcClient.bottom - rcClient.top
+            };
+
+            
```

Comment:
```suggestion
```
Please recheck in your code whether there is trailing whitespace (as this was the case here), and if so, please remove it.